homeassistant/components/livebox/config_flow.py

```diff
@@ -0,0 +1,178 @@
+            errors["base"] = "linking"
+
+        except Exception:  # pylint: disable=broad-except
+            LOGGER.error("Unknown error connecting with Livebox at %s", self.host)
+            errors["base"] = "linking"
+
+        try:
+            config = await box.system.get_deviceinfo()
+            self.box_id = config["status"]["SerialNumber"]
+
+        except Exception:
+            LOGGER.error("Unique ID not found")
+            return False
+
+        await self._entry_from_box()
+        return await self.async_step_options()
+
+        # If there was no user input, do not show the errors.
+        if user_input is None:
+            errors = {}
+
+        return self.async_show_form(step_id="link", errors=errors)
+
+    async def async_step_options(self, user_input=None):
+        """Step for link router."""
+
+        options = {}
+        if user_input is not None:
+            options = {CONF_ALLOW_TRACKER: user_input[CONF_ALLOW_TRACKER]}
+            return self.async_create_entry(
+                title=TEMPLATE_SENSOR.format(""),
+                data={
+                    "box_id": self.box_id,
+                    "host": self.host,
+                    "port": self.port,
+                    "username": self.username,
+                    "password": self.password,
+                    "options": options,
+                },
+            )
+
+        return self.async_show_form(
+            step_id="options",
+            data_schema=vol.Schema({vol.Optional(CONF_ALLOW_TRACKER): bool}),
+        )
+
+    async def _entry_from_box(self):
+        """Return a config entry from an initialized box."""
+
+        # Remove all other entries of hubs with same ID or host
+        same_hub_entries = [
+            entry.entry_id
+            for entry in self.hass.config_entries.async_entries(DOMAIN)
+            if entry.data["box_id"] == self.box_id
+        ]
+
+        if same_hub_entries:
+            await asyncio.wait(
+                [
+                    self.hass.config_entries.async_remove(entry_id)
+                    for entry_id in same_hub_entries
+                ]
+            )
+
+        return True
+
+
+class LiveboxOptionsFlowHandler(config_entries.OptionsFlow):
+    """Handle Livebox options."""
+
+    def __init__(self, config_entry):
+        """Initialize Livebox options flow."""
+        self.config_entry = config_entry
+        self.options = copy(config_entry.options)
+
+    async def async_step_init(self, user_input=None):
+        """Manage the Livebox options."""
+
+        if user_input is not None:
+            self.options[CONF_ALLOW_TRACKER] = user_input[CONF_ALLOW_TRACKER]
+            return self.async_create_entry(title="", data=user_input)
+
+        return self.async_show_form(
+            step_id="init",
+            data_schema=vol.Schema(
+                {
+                    vol.Optional(
+                        CONF_ALLOW_TRACKER,
```

Comment:
We should probably remove this option. The user can disable entities via the entity registry.

https://developers.home-assistant.io/docs/en/entity_registry_disabled_by.html

Why would we want to disable the whole platform as a user?

Reply:
Ok i remove option